tests/mock_network/utils.rs

```diff
@@ -460,18 +478,67 @@ pub fn add_mature_nodes(
     // First add the nodes that will be removed later. These can go into any sub-prefix.
     let temp_count = remove_count.saturating_sub(current_count);
     info!("Adding {} temporary nodes", temp_count);
-    for _ in 0..temp_count {
-        add_node_to_section(env, nodes, &prefix);
+    add_nodes_to_section_and_poll(env, nodes, &prefix, temp_count);
+
+    // Of the remaining nodes, `count0` goes to the 0-ending sub-prefix and `count1` to the
+    // 1-ending.
+    info!("Adding {} final nodes", count0 + count1);
+    add_nodes_to_subsections_and_poll(env, nodes, &prefix, count0, count1);
+
+    // Remove 16 mature nodes to trigger 16 age increments.
+    info!("Removing {} mature nodes", remove_count);
```

Comment:
assertion remove_count equals to 16 ?

Reply:
`remove_count` is [defined as `16`](https://github.com/maidsafe/routing/pull/2079/files#diff-c62a654da29f73317ed68538f16ae97bR463), so what would be the point of that?